### DISCUSSION TITLE
Auto-generate slugs in element editor HUDs

### DISCUSSION THREAD

**Discussion by robin-gauthier** [craftcms/cms#8692] [Ideas]
### Description

Whe you create a new category item, the handle is auto-generate on keydown and all the special characters are replace. But if I am associating a category field to my section and create a a category from the modal form, this functionnality isn't there and most users just copy paste the value with the specials characters.

### Steps to reproduce

1. Create a new category
2. Assign a category field to any section
3. Click to select category in the entry
4. Click create new category
5. Type some content in the title field.
6. The slug field stays empty.

### Additional info

- Craft version: 3.1.28
- PHP version: 7.2.16
- Database driver & version: MySQL 5.7.26

**Comment by KatieMFritz** [comment]
Yep
